ExtraRoles/PerformKillPatch.cs

```diff
@@ -29,7 +29,7 @@ private static void WriteGiveShieldRpc(PlayerControl medic, PlayerControl target
         {
             MessageWriter writer = AmongUsClient.Instance.StartRpcImmediately(PlayerControl.LocalPlayer.NetId,
                 (byte)CustomRPC.SetProtected, Hazel.SendOption.None, -1);
-            target.getModdedControl().Immortal = true;
```

Comment:
Not a joker-abilities feature, but the mod didn't compile since the type changed.